Tests/RunnerPart2/Runner2Test.php

```diff
@@ -476,6 +482,9 @@ public function testArtifactsDownload()
         /** @var Output $output */
         $output = $outputs[0];
         self::assertStringContainsString('value1', $output->getProcessOutput());
+        self::assertSame([
+            ['storageFileId' => $uploadedFileId],
+        ], $output->getArtifactsDownloaded());
```

Comment:
Ja myslim ze by chtelo do nejakyho existujiciho testu i doplnit jak se to chova kdyz to zadne artefakty nestahuje.

Reply:
Jo, doplnim

Reply:
done